python/import_util.py

```diff
@@ -0,0 +1,120 @@
+from datetime import datetime, date, time, timedelta
+import json as js
+from mage.export_import_util.parameters import Parameter
+import mgp
+
+
+def convert_from_isoformat(property):
+    if isinstance(property, str):
+        if str.startswith(property, Parameter.DURATION.value):
+            parsed_time = datetime.strptime(
+                property.split("(")[-1].split(")")[0], "%H:%M:%S.%f"
+            )
+            return timedelta(
+                hours=parsed_time.hour,
+                minutes=parsed_time.minute,
+                seconds=parsed_time.second,
+                microseconds=parsed_time.microsecond,
+            )
+        if str.startswith(property, Parameter.LOCALTIME.value):
+            return time.fromisoformat(property.split("(")[-1].split(")")[0])
+        if str.startswith(property, Parameter.LOCALDATETIME.value):
+            return datetime.fromisoformat(property.split("(")[-1].split(")")[0])
+        if str.startswith(property, Parameter.DATE.value):
+            return date.fromisoformat(property.split("(")[-1].split(")")[0])
+
+    return property
+
+
+def create_vertex(ctx, properties, labels):
+    vertex = ctx.graph.create_vertex()
+    vertex_properties = vertex.properties
+
+    for key, value in properties.items():
+        vertex_properties[key] = convert_from_isoformat(value)
+
+    for label in labels:
+        vertex.add_label(label)
+
+    return vertex.id
+
+
+def create_edge(ctx, properties, start_node_id, end_node_id, type, vertex_ids):
+    vertex_from = ctx.graph.get_vertex_by_id(vertex_ids[start_node_id])
+    vertex_to = ctx.graph.get_vertex_by_id(vertex_ids[end_node_id])
+    edge = ctx.graph.create_edge(vertex_from, vertex_to, mgp.EdgeType(type))
+    edge_properties = edge.properties
+
+    for key, value in properties.items():
+        edge_properties[key] = convert_from_isoformat(value)
```

Comment:
@jmatak I don't know, but do we want to have types also here on all functions? Or is it more module-dependent, some modules will have types, some won't?

Reply:
If we are talking about type checking in functions, yes!

Reply:
@katarinasupe okay, so just add type checks in functions